frontend/src/layout/navigation-3000/sidepanel/SidePanel.stories.tsx

```diff
@@ -18,6 +18,14 @@ const meta: Meta = {
         viewMode: 'story',
         mockDate: '2023-07-04', // To stabilize relative dates
     },
+    decorators: [
+        mswDecorator({
+            get: {
+                '/api/projects/:team_id/dashboard_templates/': {},
+                '/api/projects/:id/integrations': { results: [] },
+            },
+        }),
+    ],
```

Comment:
I think they are specific to this case because we load the side panel on the dashboards path with no items. If global mocks can be overridden in specific stories I see no reason not to add them there too. I couldn't find an example of how to do that though 😅